CHANGELOG.md

```diff
@@ -227,6 +227,7 @@
   - enabled showing of appropriate error information to the user along with refetch button [#910](https://github.com/eclipse-tractusx/portal-frontend/pull/910)
 - **Service Marketplace**:
   - enabled showing of appropriate error information to the user along with refetch button [#910](https://github.com/eclipse-tractusx/portal-frontend/pull/910)
+  - fixed service marketplace to display all active available servicesp[#1143](https://github.com/eclipse-tractusx/portal-frontend/issues/1143)
```

Comment:
Hi @Usmanfee  we have an continuous issue in the changelog file, the entry is at the wrong place and it's not even shown as conflict. I suggest you revert any change to the changelog and add the information which should be added to the changelog in the PR description and I add it then at the next release to the changelog file, sorry for the inconvenience.